lib/manager/gradle-wrapper/artifacts-real.spec.ts

```diff
@@ -22,8 +23,8 @@ const config: UpdateArtifactsConfig = {
   newValue: '5.6.4',
 };
 
-function readString(...paths: string[]): Promise<string> {
-  return readFile(resolve(fixtures, ...paths), 'utf8');
+function readString(path: string): Promise<string> {
+  return readLocalFile(path, 'utf8');
```

Comment:
This is wrong! We can't use the mocked FS here! Please revert.

🙈

@olegkrivtsov tests should mock our FS for the running code, but need to use fs-extra when reading fixtures.

Reply:
Hi @viceice I'm sorry, but I didn't catch. I see that `readLocalFile` is not mocked here.

https://github.com/renovatebot/renovate/blob/395cab2398615d0107aaed404e9d2188fa38a860/lib/manager/gradle-wrapper/artifacts-real.spec.ts#L8

You can see the only thing mocked here is `git`. All files are read from file system, I checked that when testing/debugging.

Reply:
I use `readLocalFile` here, but the only difference is it accepts relative paths to files instead of absolute ones.

Reply:
Please check why the snapshot has changed. Maybe i didn't clarified   #7049 enough. tests shouldn't use our fs for reading fixtures. tests should mock our fs if required instead of build-in fs or fs-extra.

Reply:
Also fixtures loading will be changed in #12638

Reply:
@viceice this is fixed in https://github.com/renovatebot/renovate/pull/13144